museum/testImport

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+if [ -z "$1" ]
+then
+    echo "Usage: $0 [nuke] [load] [report] <tableList>"
+    echo "    nuke:      reset your installation first"
+    echo "    load:      load the import mapping"
+    echo "    report:    populate the destination data reports. You probably don't want to run this without nuke"
+    echo "               You probably don't want to run this without nuke or if you have more than one table"
+    echo "    tableList: spaceDelimited list of field groups to load test data for. Examples:"
+    echo "               baseObject"
+    exit 1
+fi
+set -u
```

Comment:
Down here so I don't have to use getopts and also so bash doesn't complain about the undefined variables